content/use-amd-fsr.md

```diff
@@ -78,3 +78,12 @@ Lutris version 0.5.9 includes a toggle for AMD FSR in each game's preferences. T
 * Enable the "Enable AMD FidelityFX Super Resolution (FSR)" toggle in the options below.
 
 Then follow the "Setup your In-game Resolution" steps above, and enjoy FSR via Lutris!
+
+### Bottles
+
+Bottles from version 2021.10.14 also includes a toggle for AMD FSR in the preferences of each bottle.
```

Comment:
This sentence is a bit unclear.

Are you saying: "WINE bottles created in Bottles version 2021.10.14 and later?

Reply:
Got it, thanks!